app/views/pets/show.html.erb

```diff
@@ -7,12 +7,20 @@
     </div>
   </div>
 </section>
+<% if flash %>
+  <% flash.each do |key, message| %>
+  <div class="notification is-success">
+    <%= message %>
+  </div>
+  <% end %>
+  <% end %>
 <div class="card">
   <div class="card-content">
     <div class="content">
       <figure class="image is-128x128 pet_image">
         <img src=<%= @pet.image %> alt=<%= @pet.name %> >
       </figure>
+      <%= link_to "add to favorites", "/pets/#{@pet.id}/favorite", method: :post%>
```

Comment:
Nice job specifying that the method is post here. This is a step that is often hard for me to remember to do.